Xamarin.Forms.Core/FlexEnums.cs

```diff
@@ -0,0 +1,216 @@
+				if (value.Equals("flex-end", StringComparison.OrdinalIgnoreCase))
+					return FlexJustify.FlexEnd;
+				if (value.Equals("space-between", StringComparison.OrdinalIgnoreCase))
+					return FlexJustify.SpaceBetween;
+				if (value.Equals("space-around", StringComparison.OrdinalIgnoreCase))
+					return FlexJustify.SpaceAround;
+			}
+			throw new InvalidOperationException(string.Format("Cannot convert \"{0}\" into {1}", value, typeof(FlexJustify)));
+		}
+	}
+
+	public enum FlexPosition
+	{
+		Relative = Flex.Position.Relative,
+		Absolute = Flex.Position.Absolute,
+	}
+
+	[TypeConverter(typeof(FlexDirectionTypeConverter))]
+	public enum FlexDirection
+	{
+		Column = Flex.Direction.Column,
+		ColumnReverse = Flex.Direction.ColumnReverse,
+		Row = Flex.Direction.Row,
+		RowReverse = Flex.Direction.RowReverse,
+	}
+
+	[Xaml.TypeConversion(typeof(FlexDirection))]
+	public class FlexDirectionTypeConverter : TypeConverter
+	{
+		public override object ConvertFromInvariantString(string value)
+		{
+			if (value != null) {
+				if (Enum.TryParse(value, true, out FlexDirection aligncontent))
+					return aligncontent;
+				if (value.Equals("row-reverse", StringComparison.OrdinalIgnoreCase))
+					return FlexDirection.RowReverse;
+				if (value.Equals("column-reverse", StringComparison.OrdinalIgnoreCase))
+					return FlexDirection.ColumnReverse;
+			}
+			throw new InvalidOperationException(string.Format("Cannot convert \"{0}\" into {1}", value, typeof(FlexDirection)));
+		}
+	}
+
+	[TypeConverter(typeof(FlexAlignContentTypeConverter))]
+	public enum FlexAlignContent
+	{
+		Stretch = Flex.Align.Stretch,
+		Center = Flex.Align.Center,
+		FlexStart = Flex.Align.Start,
+		FlexEnd = Flex.Align.End,
+		SpaceBetween = Flex.Align.SpaceBetween,
+		SpaceAround = Flex.Align.SpaceAround,
+	}
+
+	[Xaml.TypeConversion(typeof(FlexAlignContent))]
+	public class FlexAlignContentTypeConverter : TypeConverter
+	{
+		public override object ConvertFromInvariantString(string value)
+		{
+			if (value != null) {
+				if (Enum.TryParse(value, true, out FlexAlignContent aligncontent))
+					return aligncontent;
+				if (value.Equals("flex-start", StringComparison.OrdinalIgnoreCase))
+					return FlexAlignContent.FlexStart;
+				if (value.Equals("flex-end", StringComparison.OrdinalIgnoreCase))
+					return FlexAlignContent.FlexEnd;
+				if (value.Equals("space-between", StringComparison.OrdinalIgnoreCase))
+					return FlexAlignContent.SpaceBetween;
+				if (value.Equals("space-around", StringComparison.OrdinalIgnoreCase))
+					return FlexAlignContent.SpaceAround;
+			}
+			throw new InvalidOperationException(string.Format("Cannot convert \"{0}\" into {1}", value, typeof(FlexAlignContent)));
+		}
+	}
+
+	[TypeConverter(typeof(FlexAlignItemsTypeConverter))]
+	public enum FlexAlignItems
+	{
+		Stretch = Flex.Align.Stretch,
+		Center = Flex.Align.Center,
+		FlexStart = Flex.Align.Start,
+		FlexEnd = Flex.Align.End,
+		//Baseline = Flex.Align.Baseline,
```

Comment:
Was this intentionally removed?

Reply:
yes, not supported by the engine

Reply:
left it commented out, so we know where we differ from the css spec